examples/kft-v2/trl-gpt2-checkpointing.ipynb

```
@@ -0,0 +1,1215 @@
+    "        os.environ['WORLD_SIZE'] = str(world_size)\n",
+    "        os.environ['LOCAL_RANK'] = str(local_rank)\n",
+    "        os.environ['MASTER_ADDR'] = master_addr\n",
+    "        os.environ['MASTER_PORT'] = master_port\n",
+    "\n",
+    "        if world_size > 1:\n",
+    "            try:\n",
+    "                torch.distributed.init_process_group(\n",
+    "                    backend='gloo',\n",
+    "                    rank=global_rank,\n",
+    "                    world_size=world_size\n",
+    "                )\n",
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)